src/features/profile/index.tsx

```diff
@@ -47,7 +47,7 @@ const Profile: React.FC<ProfileProps> = ({ id = undefined }: ProfileProps) => {
         setLoading(false);
       });
     }
-  }, [id, isEditing]);
+  }, [isEditing]);
```

Comment:
I believe we should keep `id` as a trigger for this reloading. Otherwise, from memory, going to another users profile and then clicking on your own will not load the change.